doc/installing.md

```diff
@@ -127,6 +129,60 @@ make check
 sudo make install
 ```
 
+### Installing blst
```

Comment:
General question, is this a hard requirement to run db-sync moving forwards?

Reply:
Yes it doesn't build otherwise. This dependency comes from `cardano-crypto-class`

Reply:
Thanking you both 👍